modules/apps/headless/headless-builder/headless-builder-impl/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.4.0"
 	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
 	compileOnly project(":apps:headless:headless-builder:headless-builder-api")
+	compileOnly project(":apps:headless:headless-delivery:headless-delivery-api")
```

Comment:
It would be great not to depend on headless-delivery. However we've decided to leave it for now as this pattern is used in other modules and it is not easy to refactor and move the creator to a separate module.

@sergiojimcos can you create a `headless-tech-debt` task for this?

Reply:
Sure!

Reply:
Here it is https://liferay.atlassian.net/browse/LPS-201191